osu.Game.Tests/Visual/UserInterface/TestSceneModSelectOverlay.cs

```diff
@@ -466,6 +553,8 @@ public void TestUnimplementedModIsUnselectable()
             waitForColumnLoad();
 
             AddAssert("unimplemented mod panel is filtered", () => getPanelForMod(typeof(TestUnimplementedMod)).Filtered.Value);
+
+            AddStep("disable panel filtering", () => getPanelForMod(typeof(TestUnimplementedMod)).Filtered.Value = false);
```

Comment:
once again should not be here ._.

Reply:
I'm confused, is this a note to yourself, or to reviewers/me?

Reply:
Just marking so I can fix it in the new pass of review.